proto/src/determined/trial/v1/trial.proto

```diff
@@ -148,3 +151,78 @@ message TrialEarlyExit {
   // The reason for the exit.
   ExitedReason reason = 1;
 }
+
+// The training metrics for some duration of training.
+message TrainingMetrics {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: {
+      required: [ "trial_id", "trial_run_id", "batches_trained", "metrics" ]
+    }
+  };
+  // The trial associated with these metrics.
+  int32 trial_id = 1;
+  // The trial run associated with these metrics.
+  int32 trial_run_id = 2;
+  // The number of batches trained on when these metrics were reported.
+  int32 batches_trained = 3;
+  // The number of batches trained on when these metrics were reported.
+  int32 records_trained = 4;
```

Comment:
why not add an optional epochs trained as well?  That is the third x-axis that would be very natural in our current system.